it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/StartControllerISpec.scala

```diff
@@ -56,7 +63,7 @@ class StartControllerISpec extends BaseControllerISpec {
       val result = await(controller.showNonAgentNextSteps(authenticatedRequest()))
 
       status(result) shouldBe 200
-      bodyOf(result) should include("/signed-out")
+      bodyOf(result) should include("/redirect-to-sos")
```

Comment:
The test this is part of is "allow the government gateway URL to be configured". Not sure if the name of this test still makes sense.